databricks_cli/configure/cli.py

```diff
@@ -35,41 +35,42 @@
 PROMPT_TOKEN = 'Token' #  NOQA
 
 
-def _configure_cli_token():
+def _configure_cli_token(profile):
```

Comment:
does it support consolidated shard? I think we need to set workspaceId for it.

Reply:
Yeah it should. The profile here is just to distinguish between different types of configurations. Since the DEFAULT profile already works with consolidated I don't see why other profiles wouldn't.